cfme/utils/appliance/__init__.py

```diff
@@ -966,6 +965,11 @@ def swap(self):
 
     def event_listener(self):
         """Returns an instance of the event listening class pointed to this appliance."""
+        # There is no REST API for event streams on versions < 5.9
+        if self.version <= '5.9':
+            from cfme.utils.events_db import EventListener
```

Comment:
**optional** for better follow trough in debugging and importing i'd like to see those renamed to `DbEventListener` and `RestEventListener`